app/views/unattended/provisioning_templates/bootdisk/generic_efi_host.erb

```diff
@@ -7,7 +7,7 @@ description: |
   Boot disk Grub2 EFI - generic host
 require:
 - plugin: foreman_bootdisk
-￼ version: 20.0.0
```

Comment:
ChatGPT explanation:

> The strange character you're seeing, "�" or "￼", is known as the Replacement Character or Object Replacement Character.
>
> Object Replacement Character (￼): In Unicode, it's represented as U+FFFC. It stands for "Object Replacement Character" and is often seen when non-textual objects (like images or symbols) are replaced.
>
> Possible Causes
> Copy-Pasting Issues:
> When text is copied from a document or website with non-standard encoding and pasted into code, the character might be inserted.
> Editor Encoding:
> Different editors may use different default encodings. If one editor writes files in one encoding (e.g., UTF-16), and another editor tries to read it in another encoding (e.g., UTF-8), you might see replacement characters.
Encoding Mismatch:
If files are transferred or read using different encodings than the original, the data might not translate correctly.
Corrupted Files:
Binary or other non-textual data can sometimes end up in text files due to corruption.